src/passes/PostEmscripten.cpp

```diff
@@ -317,7 +316,7 @@ struct PostEmscripten : public Pass {
         // The first operand is the function pointer index, which must be
         // constant if we are to optimize it statically.
         if (auto* index = curr->operands[0]->dynCast<Const>()) {
-          size_t indexValue = index->value.getInteger();
```

Comment:
This is just a drive by fix, that I noticed while fixing the line above.

Function pointer are also unsigned values.